__tests__/screens/Settings/Settings.test.js

```diff
@@ -69,4 +81,21 @@ describe('Settings page should run correctly', () => {
     });
     expect(mockNavigate).toHaveBeenCalled();
   });
+
+  it('onLogout should run correctly', () => {
+    const {getByTestId} = render(<Settings />, {wrapper: Store});
+    act(() => {
+      fireEvent.press(getByTestId('logout'));
+    });
+    expect(mockLogout).toHaveBeenCalled();
+    expect(mockStartupValue).toHaveBeenCalled();
+  });
+
+  it('delete account should run correctly', () => {
+    const {getByTestId} = render(<Settings />, {wrapper: Store});
+    act(() => {
+      fireEvent.press(getByTestId('delete'));
+    });
+    expect(showDeleteAccountAlert).toHaveBeenCalled();
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These new tests are checking if the correct functions are called when certain actions are performed. This is a good way to ensure that your components are interacting correctly with your hooks. 

```diff
+  it('onLogout should run correctly', () => {
+    const {getByTestId} = render(<Settings />, {wrapper: Store});
+    act(() => {
+      fireEvent.press(getByTestId('logout'));
+    });
+    expect(mockLogout).toHaveBeenCalled();
+    expect(mockStartupValue).toHaveBeenCalled();
+  });
+
+  it('delete account should run correctly', () => {
+    const {getByTestId} = render(<Settings />, {wrapper: Store});
+    act(() => {
+      fireEvent.press(getByTestId('delete'));
+    });
+    expect(showDeleteAccountAlert).toHaveBeenCalled();
+  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->